bot/cogs/bot.py

```diff
@@ -60,6 +65,39 @@ def __init__(self, bot: AutoShardedBot):
 
         await ctx.invoke(self.info)
 
+    async def on_message(self, msg: Message):
+        if msg.channel.id in self.allowed:
+            if self.allowed[msg.channel.id]-time.time() > 300:
+                if msg.content.count("\n") >= 3:
+                    try:
+                        tree = ast.parse(msg.content)
+
+                        # Attempts to parse the message into an AST node.
+                        # Invalid Python code will raise a SyntaxError.
+                        if not all(isinstance(node, ast.Expr) for node in tree.body):
+
+                            # We don't want multiple lines of single words,
+                            # They would be syntactically valid Python but could also be
+                            # Just some random multiline text someone is sending.
+                            howto = """Please use syntax highlighted blocks, as it makes it more legible for other users.
+
+To do this, you should input your content like this:
+
+\`\`\`python
+print("Hello world!")
+\`\`\`
+
+This will result in the following:
+```
+print("Hello world!")
+```
+"""
```

Comment:
I hate this method, because of the indentation. It just looks messy to me. Use an implicitely wrapped string: 

```py
    howto = ("Please use syntax highlighted blocks, as it makes it more legible for other users.\n"
             "\n"
             "To do this, you should input your content like this:\n"
             "... and so on"
    )
```

If someone violently disagrees with this, okay fine I'll budge. but that's my personal preference for this somewhat tricky readability problem.

Reply:
well it causes some line length errors

Reply:
yep, you still can't exceed 120. If you're in danger of doing that, simply wrap that line onto the next line and don't put in the newline.